deployment/ci-cd-license-key.md

```diff
@@ -20,8 +20,9 @@ The Telerik license activation process in CI/CD environments involves the follow
 
 1. Go to the [License Keys page](https://www.telerik.com/account/your-licenses/license-keys) in your Telerik account and download your license key.
 1. Set an environment variable with either of the following names:
-  * `TELERIK_LICENSE`&mdash;the value must be the Telerik license key string.
-  * `TELERIK_LICENSE_PATH`&mdash;the value must be the full path to the license key file, including the license file name itself. `TELERIK_LICENSE_PATH` requires `Telerik.Licensing` version `1.4.9` and above. You can use it with Telerik UI for Blazor `8.1.0` and above.
+    * `TELERIK_LICENSE`&mdash;the value must be the Telerik license key string.
+    * `TELERIK_LICENSE_PATH`&mdash;the value must be the full path to the license key file, including the license file name itself. `TELERIK_LICENSE_PATH` requires `Telerik.Licensing` version `1.4.9` and above. You can use it with Telerik UI for Blazor `8.1.0` and above.
+1. (optional) [Fail the build and deployment](#abort-deployment-on-license-key-error) if there is an issue with the license key.
```

Comment:
Isn't it expected that the next release will do this anyway? Shouldn't the build fail if there is an issue with the license key? Right now, there is no requirement to fail the build, as no watermarks are enforced, and with the next release, the build will fail anyway if there is a license issue. So why do you need this section?

Reply:
Depending on the license type, there are cases when the build will complete successfully.